topic_folders/communications/guides/index.rst

```diff
@@ -7,8 +7,9 @@ HOW-TO GUIDES
    :titlesonly:
    :glob:
 
-   
+
    adhere-style-guide.md
    submit_blog_post.md
    select-blog-tags.md
-   submit-news-item.md
\ No newline at end of file
+   submit-job-posting.md
```

Comment:
suggesting share-opportunities.md or similar to capture the varied nature of opportunities that people can share under this how-to guide